### ISSUE TITLE
data refuge project

### ISSUE THREAD

**Issue by afredston** [eco-data-science/eco-data-science-old-site#34]
Has anyone heard of or attended the recent [Data Refuge](http://www.ppehlab.org/) events that bring together programmers to scrape and safely store government data at risk from the Trump administration (recently featured by [NYT](https://www.nytimes.com/2017/03/06/science/donald-trump-data-rescue-science.html))? Would anyone be interested in co-organizing one in Santa Barbara starting with the eco-data-science community?

**Comment by mbjones** [comment]
@afredstonhermann Our local DataONE colleagues wrote a newsletter article about it last month ([Crowd Sourcing the Harvest of Federal Data](https://www.dataone.org/sites/default/files/sites/all/documents/newsletters/dataonenews_winter2016-7_sm_0.pdf)), and I think @hsoyka and @aebudden were discussing a workshop in Santa Barbara. Not sure where that ended up.

And, related, the NYT published an interesting [article about data rescue](https://www.nytimes.com/2017/03/06/science/donald-trump-data-rescue-science.html?_r=2) efforts and the recent NCEI announcements.

**Comment by bbest** [comment]
Yes @afredstonhermann @mbjones, meeting up in Santa Barbara around [Data Refuge.org](https://www.datarefuge.org) / #datarescue sounds awesome! Chatted about this with: 
- @brucecaron of [ESIP](http://esipfed.org) fame and the possibility of peer-to-peer distribution a la [BitTorrent](http://www.bittorrent.com/) (@JamesFrew thought of "opendapster" a while back)
- @LucaFoschini, coordinator of [Santa Barbara Data Science Meetup](https://www.meetup.com/Santa-Barbara-Data-Science/)
- @stevejbrown, coordinator of [UCSB Data Science](http://ucsb-data-science.github.io/)

I can advertise on [SB R Users Meetup](https://www.meetup.com/Santa-Barbara-R-Users-Group/), which @brunj7 or @cjlortie can also do. I'll be out of town March 13 - April 1, but please proceed if y'all can meet.

**Comment by aebudden** [comment]
DataONE as a project isn't hosting / sponsoring any events but I think @hsoyka and other at NCEAS are interested in a community event.
